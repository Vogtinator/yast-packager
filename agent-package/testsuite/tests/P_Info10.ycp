{
    return SCR::Execute(.setEnvironment,$["packageinfopath":"tests","language":"en","common.pkd":"common.kernel.pkd","dudir":"tests/du.dir","partition":[$["name":"/","used":0,"free":10000],$["name":"bin","used":0,"free":10000],$["name":"etc","used":0,"free":10000],$["name":"sbin","used":0,"free":10000],$["name":"usr","used":0,"free":10000],$["name":"var","used":0,"free":10000]]]);
}
{
    return SCR::Read(.kernelList);
}
