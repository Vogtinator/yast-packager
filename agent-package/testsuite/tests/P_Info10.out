(true)
($["k_deflt":"The standard SuSE kernel", "k_deflt_22":"The standard SuSE kernel", "k_i386":"Kernel compiled for i386", "k_i386_22":"Kernel compiled for i386", "k_orig":"Kernel from pristine Linux sources", "k_psmp":"Kernel 2.4 with SMP-support for older processors (Pentium classic)", "k_smp":"Kernel with multiprocessor support", "k_smp_22":"Kernel with multiprocessor support"])
